Microsoft.Azure.CosmosRepository/test/Stubs/TestItemWithEtag.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Azure.CosmosRepository;
+
+namespace Microsoft.Azure.CosmosRepositoryTests.Stubs
+{
+    public class TestItemWithEtag : Item
```

Comment:
Where is the ETag here?

Reply:
It is currently in Item, I will be updating in line with the discussion below.